src/libs/ReportUtils.js

```diff
@@ -790,6 +789,7 @@ function getIcons(report, personalDetails, defaultIcon = null, isPayer = false)
         }
```

Comment:
Just to be clear, you're not adding `id` to the above sections (`isDomainRoom`, `isAdminRoom`, `isAnnounceRoom`) because we currently don't support custom avatars in those reports?

Reply:
Correct. Those are static avatars yet.